File: Facebook.NET/Contracts/GraphAPI.Venue.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphApi
    {
        [DataContract]
        public sealed class Venue
        {
            [DataMember(Name = "id")]
            public string ID { get; set; }

            [DataMember(Name = "street")]
            public string Street { get; set; }

            [DataMember(Name = "city")]
            public string City { get; set; }

            [DataMember(Name = "state")]
            public string State { get; set; }

            [DataMember(Name = "zip")]
            public string Zip { get; set; }

            [DataMember(Name = "country")]
            public string Country { get; set; }

            [DataMember(Name = "latitude")]
            public string Latitude { get; set; }

            [DataMember(Name = "longitude")]
            public string Longitude { get; set; }
        }
    }
}

File: Facebook.NET - VB/Contracts/GraphAPI.Venue.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class Venue
            <DataMember(Name:="id")>
            Public Property ID() As String
            <DataMember(Name:="street")>
            Public Property Street() As String
            <DataMember(Name:="city")>
            Public Property City() As String
            <DataMember(Name:="state")>
            Public Property State() As String
            <DataMember(Name:="zip")>
            Public Property Zip() As String
            <DataMember(Name:="country")>
            Public Property Country() As String
            <DataMember(Name:="latitude")>
            Public Property Latitude() As String
            <DataMember(Name:="longitude")>
            Public Property Longitude() As String
        End Class
    End Class
End Namespace